scripts/generate-tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
where is this script run? Just once to generate the "original" results I suppose?

Reply:
Yep. It should only be rerun, if we want to change our expected (baseline) rulesets.